### DISCUSSION TITLE
Proposal: add 'restart: always' to all services in the docker-compose file

### DISCUSSION THREAD

**Discussion by Verhoeckx** [uxbox/uxbox#2928] [Ideas]
Hello Penpot developers,

First: thank you for this fantastic application!

When running Penpot on the server, I notice that only three of the six services in the docker-compose file contain the attribute `restart: always`. The result of this, is that only those services are restarted after (re-)booting up the server.

My question: shouldn't all the services in the docker-compose file contain the attribute `restart: always`?
I think they do and would like to propose to add the attribute `restart: always` to _all_ services.

<br>

Below you can find which services are started automatically and which don't.

Services/containers that are started automatically:
`penpot-redis`  (contains `restarts: always`)
`penpot-postgres` (contains `restarts: always`)
 `penpot-mailcatch` (contains `restarts: always`)

Services/containers that are _not_ started automatically:
`penpot-exporter`
`penpot-backend`
`penpot-frontend` 

<br>

Let me know what you think of this proposal.

**Comment by niwinz** [comment]
Thanks, I will consider your suggestion. You are pretty right that they all should have `always` or `unless-stopped`. Next time,, please open an issue or ask on community, discussions are deprecated and we do not guarantee answering here.

**Reply by Verhoeckx** [reply]
Thanks and next time I will create an issue on the community forum!

**Reply by niwinz** [reply]
FYI, the changes are already applied on the `main` branch`, thanks for the suggestion

**Reply by Verhoeckx** [reply]
Ah, that's great news!

I just used the new docker-compose file, rebooted the server and all services came up well! Thanks!

And to be honest, I first reported the issue on the community forum ([here](https://community.penpot.app/t/start-penpot-automatically-on-boot/2467)), but because I didn't get the desired response, I reported it here again. I didn't know however, that the discussions aren't used anymore.

**Reply by niwinz** [reply]
sorry for the possible confusion :smile:

**Reply by niwinz** [reply]
next time, if you have is an enhancement or  you found a possible bug, directly open issue on GitHub (not discussions, issues hehe)

**Reply by Verhoeckx** [reply]
Will do that :smiley: !
